modules/priors/NAMESPACE

```diff
@@ -1,15 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
-S3method(plot,posterior.density)
-S3method(plot,prior.density)
-S3method(plot,trait)
```

Comment:
@dlebauer Tried running this locally and my version of Roxygen wants to re-add these S3 method declarations. How did you persuade it to treat these as regular functions?